doc/contributing/security-release-process.md

```diff
@@ -124,7 +124,7 @@ out a better way, forward the email you receive to
 `oss-security@lists.openwall.com` as a CC.
 
 * [ ] Post in the [nodejs-social channel][]
-  in the OpenJS slack asking for amplication of the blog post.
+  in the OpenJS slack asking for reach amplification of the blog post.
```

Comment:
I think this could just left left as `amplification1 versus `reach applification`